cmake/Modules/CodeCoverage.cmake

```diff
@@ -362,14 +353,13 @@ function(ADD_CODE_COVERAGE)
         COMMAND ${LCOV_PATH} ${LCOV_EXTRA_FLAGS} --directory . --capture --output-file ${PROJECT_BINARY_DIR}/${Coverage_NAME}.info
                 || echo "WARNING: No cpp report to output"
         # add baseline counters
-        COMMAND ${LCOV_PATH} -a ${PROJECT_BINARY_DIR}/${Coverage_NAME}.base -a ${PROJECT_BINARY_DIR}/${Coverage_NAME}.info
+        COMMAND ${LCOV_PATH} -a ${PROJECT_BINARY_DIR}/${Coverage_NAME}_base_cpp.info -a ${PROJECT_BINARY_DIR}/${Coverage_NAME}.info
```

Comment:
Please add it to `OUTPUT`.
I think some other files are missing from `OUTPUT`.

Reply:
I change to use `add_custom_command` and `add_cutom_target` to add `output`.